analysis/cf_trf_estimate_example.py

```diff
@@ -0,0 +1,281 @@
+"""
```

Comment:
I used the prefix "cf" (stands for "cedric foucault") because it helps me quickly find my code while I'm working on it, but we can discuss if we want a better name for this script (and same for the `utils.py` module below).